plugins/postcss-media-minmax/test/env.expect.css

```diff
@@ -0,0 +1,7 @@
+@media (min-height: calc((safe-area-inset-top) + 0.02px)) {}
```

Comment:
Just curious here, why is it 0.02 and not 0.01 ? Is that the limit?

Reply:
`0.02` is the smallest fraction that works well for some older Safari versions.
This is something that was tested by bootstrap and reported to `postcss-media-minmax`.

Anything smaller than `0.02` is rounded away.
So `900.01 === 900` but `900.02 !== 900`

Reply:
https://github.com/postcss/postcss-media-minmax/pull/28#issue-800876301

Reply:
Thanks a lot!!